docs/sources/k6/next/extensions/explore.md

```diff
@@ -9,12 +9,10 @@ weight: 01
 
 Extensions can be used to extend the core k6 features to fit your performance-testing use cases. There are two main categories of extensions:
 
-| Category      | Maintainer   | Audited | Cloud support |
-| ------------- | ------------ | ------- | ------------- |
-| **Official**  | Grafana Labs | ✅ Yes  | ✅ Yes        |
-| **Community** | Community    | ✅ Yes  | ⚠️ Partial\*  |
-
-\* _Partial_ cloud support means that the extensions can be used in the CLI with the `--local-execution` mode, and on Private Load Zones with a custom image. They don't work on Grafana Cloud k6 Public Load Zones.
+| Category      | Maintainer   | Audited | Cloud support | Guaranteed updates |
```

Comment:
What do we mean by "Guaranteed updates" here @Lantero? 🤔 

I'm wondering if this might be better explained outside of a column.

Reply:
Important security patches/backports mostly. Any thoughts on how we could incorporate this information? It is an important (the main) difference between official and community. With community, we vet a specific version and "forget" i.e. we don't take responsibility for keeping it up to date, you should use them at your own risk and rely on the maintainers for these type of patches.

Without it, the original table would look weird since it would be the same for both official and community, except for ownership. We could also challenge whether we need this table or not, now that both official and community are cloud supported.

Reply:
5099f5589b324ccda9f43bd24dce3d389910077e